cli/createTemplate.sh

```diff
@@ -756,25 +762,25 @@ function process_template() {
   #
   # The cleanup logic for >=2.0.1 cmdb is also more robust, as it will remove files
   # that are no longer generated.
-  local deployment_unit_state_subdirectories="false"
-  case "${level}" in
-    unitlist|blueprint|buildblueprint)
-      # No subdirectories for deployment units
-      ;;
-    *)
-      if [[ -d "${cf_dir_default}" ]]; then
+  if [[ -z "${OUTPUT_DIR}" ]]; then
+    local deployment_unit_state_subdirectories="false"
+    case "${level}" in
+      unitlist|blueprint|buildblueprint)
+        # No subdirectories for deployment units
+        ;;
+      *)
         readarray -t legacy_files < <(find "${cf_dir_default}" -mindepth 1 -maxdepth 1 -type f -name "*${deployment_unit}*" )
```

Comment:
The existence check for cf_dir_default has been reverted - I just added it to avoid a find error. Please re-instate

Reply:
I rebased the PR to include that specifically, not sure where it went sorry